src/Arcus.Messaging.Pumps.ServiceBus.KeyRotation/Arcus.Messaging.Pumps.ServiceBus.KeyRotation.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageId>Arcus.Messaging.Pumps.ServiceBus.KeyRotation</PackageId>
+    <Authors>Arcus</Authors>
+    <Company>Arcus</Company>
+    <Product>Arcus.Messaging</Product>
+    <Description>Provides capability related to key rotation of Azure Service Bus message pumps</Description>
```

Comment:
```suggestion
    <Description>Provides capability related to the resiliency of Azure Service Bus message pumps in case of key rotation</Description>
```